fixtures/web-first/text/geometry/add_case.py

```
@@ -105,16 +124,18 @@ def main() -> None:
         or not 1 <= units_per_em <= 65535
     ):
         sys.exit("refused: units_per_em must be a positive 16-bit integer")
-    if not isinstance(facts["glyphs"], list) or len(facts["glyphs"]) != len(content):
-        sys.exit("refused: rung B requires one direct cmap glyph fact per ASCII byte")
-    for index, (character, glyph) in enumerate(zip(content, facts["glyphs"])):
+    if not isinstance(facts["glyphs"], list) or len(facts["glyphs"]) != len(scalars):
+        sys.exit("refused: rung B requires one direct cmap glyph fact per source scalar")
+    source_utf8_byte = 0
+    source_utf16_index = 0
+    for index, (character, glyph) in enumerate(zip(scalars, facts["glyphs"])):
```

Ruff (0.16.3)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)